### DISCUSSION TITLE
Welcome to ncxprogramming.com Discussions!

### DISCUSSION THREAD

**Discussion by NinjaCheetah** [NCX-Programming/site#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  If you have any questions or suggestions about our site, feel free to talk about them here.

**Comment by rvtr** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Ok then. I'm Ian, I like the C64, and I help out with the website sometimes.
